projects/Mallard/src/hooks/use-edition-provider.tsx

```diff
@@ -141,7 +142,10 @@ export const getEditions = async (
             // Grab editions list from the endpoint
             const editionsList = await fetchEditions(apiUrl)
             if (editionsList) {
-                const filteredList = removeExpiredSpecialEditions(editionsList)
+                const showAllEditions = await showAllEditionsCache.get()
+                const filteredList = showAllEditions
```

Comment:
Do we want a switch just to show/hide special edition?

Reply:
hmm I'm not sure, I think this could be global for all editions - it looks like we'll be adding new behaviour soon so that if an Edition has 0 issues then we hide it from the menu - which could apply to special editions as well as regional ones.